vite.config.ts

```diff
@@ -140,8 +140,23 @@ export default defineConfig(({ mode }) => {
           // https://stackoverflow.com/a/72440811/10752354
           rollupPolyfillNode(),
         ],
+        output: {
+          manualChunks(id) {
+            // Create one different chunk for each node_module
+            if (id.includes('node_modules')) {
+              const chunkName = id
+                .toString()
+                .split('node_modules/')[1]
+                .split('/')[0]
+                .toString();
+              return chunkName;
+            }
+          },
```

Comment:
I guess this is what seems dangerous. Creates a massive number of chunks. Do we really need to chunk every node_module?

Reply:
yeah, there is more requests now 
<img width="236" alt="Screenshot 2023-10-10 at 18 52 36" src="https://github.com/balancer/frontend-v2/assets/46521087/5150e211-efd3-4aba-9ed7-492f7171f6ba">

Reply:
Yeah, we can customize this function to our needs. For example, only creating explicit chunks for certain bigger node_modules like balancer-sdk and leaving the default behaviour/grouping for other smaller node_modules.

That's why I told @anastas to take a look together.